travis/deploy_pull_request.sh

```diff
@@ -39,9 +39,9 @@ function publish_pull_request_urls_in_github {
   else
     REVIEW_FEATURE_MESSAGE="\
 {\"body\": \"Travis automatic deployment:\\n \
-  * **🔭 [Explorer Dapp]($REVIEW_FEATURE_URL)**: Explorer test app\\n \
-  * **📈 [Trade Dapp]($REVIEW_FEATURE_URL/trade.html)**: Trade test app\\n \
-  * **📚 [Storybook](${REVIEW_FEATURE_URL}/storybook/)**: Component stories\""
+  * **🔭 [Explorer Dapp](${{ REVIEW_FEATURE_URL }})**: Explorer test app\\n \
+  * **📈 [Trade Dapp](${{ REVIEW_FEATURE_URL }})**: Trade test app\\n \
```

Comment:
Intentional that both point to the same URL?

Reply:
not at all... will fix

Reply:
mm, that's odd, I think u reviewed on old code or sth. it should look like this

![image](https://user-images.githubusercontent.com/2352112/104049450-47450b80-51e5-11eb-8872-89664e993a79.png)

Reply:
o.O

Reply:
Nope, that's the diff gh is giving me still
![screenshot_2021-01-08_10-51-39](https://user-images.githubusercontent.com/43217/104053079-7fc1f680-519f-11eb-90d4-cb6e5e359559.png)